pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/PersistentTopics.java

```diff
@@ -1016,6 +1016,9 @@ public Response peekNthMessage(@PathParam("property") String property, @PathPara
                 responseBuilder.header("X-Pulsar-publish-time",
                         DATE_FORMAT.format(Instant.ofEpochMilli(metadata.getPublishTime())));
             }
+            if (metadata.hasNumMessagesInBatch()) {
+                responseBuilder.header("X-Pulsar-num-msg-batch", metadata.getNumMessagesInBatch());
```

Comment:
should it be `X-Pulsar-num-batch-message`?

Reply:
Changed